draft-ietf-rats-architecture.md

```diff
@@ -610,6 +610,19 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.  The component that is
 implicitly trusted is often referred to as a Root of Trust.
 
+Implicit trust can also be tied to the communications link over which
+the attestation Evidence is conveyed. As an example, the Attester and Verifier can
+communicate over a link that is established using a Root of Trust.  An example of such communicaitons
+is over a cellular link, where the link is established by the Attester leveraging a secure
+element (e.g. SIM card).  The Verifier may have esablished the communications link and verified
+the preence of the Root-of-Trust in doing so, or may have received evidence that the communications
+link is anchored to a Root of Trust from an entity that established the communications link.
```

Comment:
Need more explanation about what "anchored to a Root of Trust" means.  To trust the evidence, you have to have some assurance that the evidence is correct, and wasn't just fed to the communication link endpoint by some untrusted party.

Reply:
What about the following:

"... or may have received evidence that the communications link is anchored to a Root of Trust from an entity that established the communications link.  In the latter case, anchoring the communications link to a Root of Trust would mean that  this entity only established a communications link with the attesting device after verifying that the device had a Root of Trust."